battles/tasks.py

```diff
@@ -0,0 +1,12 @@
+import logging
+
+from battles.helpers.fight import run_battle
+from battles.models import Battle
+from pokebattle import celery_app
+
+
+@celery_app.task
+def task_run_battles(battle_id):
+    logging.info('run battles')
```

Comment:
Great initiative here! Usually, when logging on tasks we use a special logger: http://docs.celeryproject.org/en/latest/userguide/tasks.html#logging

Reply:
Thank you. Done!